ZupIT/beagle-web-react
# Feature/accessibility

**- What we did** (@jamessonpenhazup and I)
Added support to the Accessibility object that comes from BFF and also implemented accessibility attributes on beagle components (may help some disabled people)

**- How we did it** (@jamessonpenhazup and I)
Created a function that builds the new accessibility attributes for components and created a map to make easier the insertion of new attributes

**- How to verify it** (@jamessonpenhazup and I)
When you create an Accessibility object on BFF, should be `accessible: true`, provide `accessibilityLabel` with some text, and the element that you expect should come with `aria-label="your text"`. If you also provide the `isHeader: true` attribute, you should see that your element will come with the attribute defined as `role="heading"`
